packages/web-components/src/components/card-link/card-link.scss

```diff
@@ -14,6 +14,10 @@
 :host(#{$dds-prefix}-card-link),
 :host(#{$dds-prefix}-link-list-item-card),
 :host(#{$dds-prefix}-link-list-item-card-cta) {
+  .#{$prefix}--card {
+    max-width: 320px;
```

Comment:
this max-width will break the specs at larger breakpoints 
![Screen Shot 2021-04-27 at 3 18 37 PM](https://user-images.githubusercontent.com/20210594/116319709-d9a8f500-a76b-11eb-93df-eaa0d736af3c.png)
![Screen Shot 2021-04-27 at 3 19 50 PM](https://user-images.githubusercontent.com/20210594/116319821-0a892a00-a76c-11eb-8251-a5324102fb00.png)

Reply:
@IgnacioBecerra can you take a look?

Reply:
@ariellalgilmore Good catch, I'm checking with designers about defining a `max-width` for different breakpoints that could fix this. Thanks!

Reply:
thanks @IgnacioBecerra !!

Reply:
fyi: @DragosRistici has confirmed a max-width of 320px has approved by design. PR will be merged shortly :)